tests/bulk/bulkwrite-delete_error-001.phpt

```diff
@@ -17,4 +21,7 @@ echo throws(function() use ($bulk) {
 --EXPECTF--
 OK: Got MongoDB\Driver\Exception\InvalidArgumentException
 Expected "collation" option to be array or object, int%S given
+
+OK: Got MongoDB\Driver\Exception\InvalidArgumentException
+Expected "hint" option to be string, array, or object, int%S given
```

Comment:
Was `int%S` for PHP 5.x compatibility? I think PHP 7.x always uses the same term ("int" or "integer"), so we might be able to do without the `%S` and just replace these across the board.

Reply:
I'll check and replace these in a separate pull request. Good point 👍